src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +16,15 @@ public class StreamPractice {
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .map(s -> s.split(","))
```

Comment:
All magic numbers, strings, symbols in your code should be constants.
[this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)
don't use short name, fix everywhere
```suggestion
                .map(line -> line.split(SEPARATOR))
```